TUM Campus App/CalendarViewController.swift

```diff
@@ -121,7 +121,7 @@ extension CalendarViewController {
         if !sameDay(newDate, b: date) {
             newDate = date
         }
-        dayPlannerView.scroll(to: newDate, options: MGCDayPlannerScrollDateTime, animated: animated)
+        //dayPlannerView.scroll(to: newDate, options: MGCDayPlannerScrollDateTime, animated: animated)
```

Comment:
@mathiasquintero Can you have a look at this? This was the only issue I was not able to resolve, even after and hour of research. I just get this error:

> Use of unresolved identifier 'MGCDayPlannerScrollDateTime'